CameraInterface/CameraInterface.py

```diff
@@ -0,0 +1,64 @@
+##
+# @file CameraInterface.py
+# @brief カメラから画像を取得、保存を行う
+# @author kawano
+# 実行コマンド PowerShell.exe -ExecutionPolicy RemoteSigned -File captureCameraInterface.ps1 カメラ番号
+import os
+# cv2.VideoCaptureの処理時間短縮(import cv2の前に書く必要あり)
+# @see https://qiita.com/youichi_io/items/b894b85d790720ea2346
+os.environ["OPENCV_VIDEOIO_MSMF_ENABLE_HW_TRANSFORMS"] = "0"
+import cv2
+import sys
+
+# カメラ仲介クラス
+class CameraInterface:
+    ##
+    # @brief コンストラクタ
+    # @param cameraId カメラ番号
+    def __init__(self, cameraId):
+        # 画像取得するカメラを選択する（引数はカメラ番号）
+        self.camera = cv2.VideoCapture(cameraId)
+
+    ##
+    # @brief カメラから画像を取得、保存を行う
+    def getFrame(self):
+        # successed   : 画像が取得が成功したか(True or False)
+        # frame : 画像
+        successed, frame = self.camera.read()
+        if successed:
+            # 画像の保存
+            cv2.imwrite("course.png", frame)
+            print("画像を保存しました")
+        else:
+            print("画像を取得できませんでした")
+
+    ##
+    # @brief カメラが接続されているかどうかを一覧表示
+    @staticmethod
+    def checkCameraConnection():
+        print('接続されているカメラの番号を調べています...')
+        trueCameras = []  # 空の配列を用意
+
+        # カメラ番号を0～9まで変えて、COM_PORTに認識されているカメラを探す
+        for cameraNumber in range(0, 10):
+            try:
+                camera = cv2.VideoCapture(cameraNumber)
+                successed, frame = camera.read()
```

Comment:
return値を無視する場合は、`_`を使用した方が良いようです。
```suggestion
                successed, _ = camera.read()
```

参考: https://qiita.com/kiii142/items/6879cb065ad4c5f0b901#return%E5%80%A4%E3%82%92%E7%84%A1%E8%A6%96%E3%81%99%E3%82%8B%E3%81%9F%E3%82%81%E3%81%AE_